tox.ini

```diff
@@ -82,6 +88,8 @@ commands =
 [coverage:run]
 branch = True
 source = zope.sqlalchemy
+omit =
+    */tests.py
```

Comment:
```suggestion
```

Tests should always be part of coverage, this allows to see which code in the tests is not executed and can be deleted.

Reply:
The issue with that is, that the coverage part of the tests only covers part of the tests so coverage falls below 70%. You would need to calculate coverage for every SQLAlchemy version (and also do the various postgres specific test runs) and stitch the results together to get a more accurate picture.

This would have been a large refactor of the test workflow on GitHub actions and it is definitely something you should try to do in the future, but it seemed out of scope for this PR, this seemed like the most sensible band-aid solution in the meantime, since you are not accurately measuring that part of the test coverage anyways, so it seems sensible to omit it until it is measured closer to correct.

Reply:
I have opeened an issue for this so we can track this it separately: #80

Reply:
I have no problem with coverage dropping below 70 %. This can be solved in #80 using combined coverage. But here I'd like to keep the actual coverage values over the whole code in this package.